Dockerfile

```diff
@@ -66,8 +72,24 @@ RUN apt-get update \
         libscrypt-dev=$SCRYPT_VERSION \
         libmpfr-dev=$MPFR_VERSION \
         libgmp3-dev=$GMP3_VERSION \
+        pegtl-dev=$PEGTL_VERSION \
+        rapidjson-dev=$RAPIDJSON_VERSION \
+        libgtest-dev=$GTEST_VERSION \
     && rm -rf /var/lib/apt/lists/*
 
+# Install cppgraphqlgen
+RUN curl -SL https://github.com/microsoft/cppgraphqlgen/archive/v$CPPGRAPHQLGEN_VERSION.tar.gz \
+    | tar -xzC /tmp \
+    && curl -SL https://github.com/taocpp/PEGTL/archive/7d039707cf835cea63daa78a717e18fcc5bcf95b.tar.gz \
+    | tar -xzC /tmp/cppgraphqlgen-${CPPGRAPHQLGEN_VERSION} \
```

Comment:
there is also official pegtl-dev ubuntu package, not sure what's the deal with versions if that would be suitable but if yes, might be easier and cleaner to use official package

Reply:
Actually I did add the official with this line:
pegtl-dev=$PEGTL_VERSION \

I should have actually remove that since I added it this way.
For some reason with that official one I am getting some errors. It only work with this specific version which is a submodule there

Reply:
hmm ok, and did you also consider to download it through submodule init ?

Reply:
I guess I would need then to download cppgraphqlgen with git clone. I think if you download a zipped version there is no git submodule info in there or am I mistaken?

Reply:
If you have a cleaner way to do it, please do. I know that this downloading and then copying is not really a nice way to do it :)

Reply:
Oh yeah you are right, you do not have git info when downloading .gz file... Did not realize that :)